RallyHereIntegration/Source/RallyHereIntegration/Private/RH_LocalPlayerSessionSubsystem.cpp

```diff
@@ -892,25 +902,45 @@ void URH_LocalPlayerSessionSubsystem::OnPlatformSessionDestroyed(FName SessionNa
 	}
 }
 
-void URH_LocalPlayerSessionSubsystem::OnPlatformSessionParticipantChange(FName SessionName, const FUniqueNetId& UniqueNetId, bool bJoined)
+void URH_LocalPlayerSessionSubsystem::OnPlatformSessionParticipantsChanged(FName SessionName, const FUniqueNetId& UniqueNetId, bool bJoined)
 {
-	auto* Syncer = GetPlatformSyncerByRHSessionId(SessionName.ToString());
-	if (Syncer != nullptr)
+	if (!bJoined)
```

Comment:
Stylistically, we generally prefer to early out, rather than nesting if statements so deep.  Not a serious issue

Reply:
I think there was originally an else case here for trying to handle your own player leaving, but it was removed because that is bound elsewhere.  It may still be useful later on, though generally I agree on the stylistic element.